fastseq_cli/transformers_generate.py

```diff
@@ -118,6 +247,24 @@ def run_generate():
     parser.add_argument("--without_fastseq_opt", action="store_true")
     parser.add_argument("--no_repeat_ngram_size", type=int, default=None,
                          required=False, help="size of no repeat ngram")
+    parser.add_argument("--include_special_tokens", action="store_true")
+    parser.add_argument("--clean_up_tokenization_spaces", action="store_true")
+    parser.add_argument("--preprocess_workers",
+                        type=int,
+                        default=2,
+                        required=False,
+                        help="pre-processing worker threads")
+    parser.add_argument("--postprocess_workers",
+                        type=int,
+                        default=2,
```

Comment:
in fairseq, default value is 1. Different from here. Can we unify them? It impacts benchmarks, please study and select a good value for both.